starlette_admin/statics/i18n/dt/de.json

```diff
@@ -0,0 +1,111 @@
+{
+	"aria": {
+		"sortAscending": ": aktivieren, um Spalte aufsteigend zu sortieren",
+		"sortDescending": ": aktivieren, um Spalte absteigend zu sortieren"
+	},
+	"buttons": {
+		"collection": "Sammlung <span class='ui-button-icon-primary ui-icon ui-icon-triangle-1-s'/>",
+		"colvis": "Spalten anzeigen/ausblenden",
+		"colvisRestore": "Sichtbarkeit wiederherstellen",
+		"copy": "Kopieren",
+		"copyKeys": "Drücken Sie Strg oder u2318 + C, um die Tabellendaten in die Zwischenablage zu kopieren.<br><br>Um abzubrechen, klicken Sie auf diese Nachricht oder drücken Sie Escape.",
+		"copySuccess": {
+			"1": "1 Zeile in die Zwischenablage kopiert",
+			"_": "%d Zeilen in die Zwischenablage kopiert"
+		},
+		"copyTitle": "In Zwischenablage kopieren",
+		"createState": "Status erstellen",
+		"csv": "CSV",
+		"excel": "Excel",
+		"pageLength": {
+			"-1": "Alle Zeilen anzeigen",
+			"_": "%d Zeilen anzeigen"
+		},
+		"pdf": "PDF",
+		"print": "Drucken",
+		"removeAllStates": "Alle Status entfernen",
+		"removeState": "Entfernen",
+		"renameState": "Umbenennen",
+		"savedStates": "Gespeicherte Status",
+		"stateRestore": "Status %d",
+		"updateState": "Aktualisieren"
+	},
+	"datetime": {
+		"amPm": [
+			"vormittags",
+			"nachmittags"
+		],
+		"hours": "Stunde",
+		"minutes": "Minute",
+		"months": [
+			"Januar",
+			"Februar",
+			"März",
+			"April",
+			"Mai",
+			"Juni",
+			"Juli",
+			"August",
+			"September",
+			"Oktober",
+			"November",
+			"Dezember"
+		],
+		"next": "Weiter",
+		"previous": "Zurück",
+		"seconds": "Sekunde",
+		"unknown": "-",
+		"weekdays": [
+			"So",
+			"Mo",
+			"Di",
+			"Mi",
+			"Do",
+			"Fr",
+			"Sa"
+		]
+	},
+	"decimal": ".",
+	"emptyTable": "Keine Daten in der Tabelle vorhanden",
+	"info": "_START_ bis _END_ von _TOTAL_ Einträgen anzeigen",
+	"infoEmpty": "0 bis 0 von 0 Einträgen anzeigen",
+	"infoFiltered": "(gefiltert von _MAX_ Gesamteinträgen)",
+	"infoThousands": ".",
```

Comment:
I am unsure about this. I think `decimal `and `infoThousands `should be different.

Reply:
yes, you're right, `decimal` divider should be comma, e.g. `1.000.000,99`